src/main/java/ai/rev/speechtotext/CustomVocabulariesClient.java

```diff
@@ -42,7 +42,7 @@ public CustomVocabulariesClient(String accessToken) {
    * submission and its progress.
    *
    * @param submission An object that contains the custom vocabularies as well as optional metadata
-   *     and callback url
+   *     and notification config
```

Comment:
I think we can remove the enumeration of the internals on the comment for this parameter entirely. No need to list the specifics of this CustomVocabularySubmission object and change it on every addition.